### PR TITLE
Translate Ruby 2.4.0 (es)

### PR DESCRIPTION
**Pull request by FerPerales**

### CODE REVIEW

**Comment by stomar** [comment]
cc @dabit

File: es/news/_posts/2016-12-25-ruby-2-4-0-released.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+---
+layout: news_post
+title: "Liberado Ruby 2.4.0"
+author: "naruse"
+translator:
+date: 2016-12-25 00:00:00 +0000
+lang: en
```
Comment (stomar) [inline es/news/_posts/2016-12-25-ruby-2-4-0-released.md]: es

**Review by gfvcastro — CHANGES_REQUESTED** [review]
Thank you for your contribution!

File: es/news/_posts/2016-12-25-ruby-2-4-0-released.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
+---
+layout: news_post
+title: "Liberado Ruby 2.4.0"
```
Comment (stomar) [inline es/news/_posts/2016-12-25-ruby-2-4-0-released.md]: However, "Liberado Ruby ..." has previously been used **very** often, too. And it has the "full name" of the version (i.e. "Ruby 2.4.0") in unmodified form.

But you must decide what sounds better in `es`.

**Comment by stomar** [comment]
And, please wrap lines at max. 80 characters, where possible.

**Comment by FerPerales** [comment]
@gfvcastro changes done. Sorry for letting this stale for so long

**Comment by stomar** [comment]
This can not be merged. **Never** merge upstream into your feature branch.
(See also https://github.com/ruby/www.ruby-lang.org/wiki#commits-and-commit-messages.)

Please rebase your feature branch onto current master or drop the merge commit and repush, or open a clean PR.

**Pull request closed without merging**